knee_sarg/assets/oai.py

```diff
@@ -287,3 +290,52 @@ def has_image_files(
     ),
     ins={"runs": AssetIn("cartilage_thickness_runs")},
 )
+
+
+class CollectImagesConfig(Config):
+    out_dir: str = Field(
+        default_factory=lambda: "all-images",
+        description="JSON file with array of patient IDs",
+    )
+    files_to_collect: List[str] = Field(
+        default_factory=lambda: THICKNESS_IMAGES,
+        description="List of files to copy into out directory",
```

Comment:
Better description: `Name of files that appear in each case's out directory that are to be copied into "all-images".`.

Reply:
yes, better, changed.